packages/manager/apps/dedicated/client/app/dedicated/server/bandwidth/public-order/public-order.controller.js

```diff
@@ -112,4 +113,17 @@ export default class {
   seeOrder() {
     this.$window.open(this.model.orderUrl, '_blank');
   }
+
+  getServiceUpgradeParams() {
+    const { duration, pricingMode } = this.plans
+      .find(({ planCode }) => planCode === this.model.plan)
+      ?.prices?.find(({ capacities }) => capacities.includes('renew'));
```

Comment:
Probably same issue like here :
https://github.com/ovh/manager/pull/8011#discussion_r977604281

issue is related to destructuring undefined.

Reply:
Done